source/server/spongineer/bugreport.rst

```diff
@@ -15,7 +15,7 @@ Whenever you report a bug, please include the following:
 * any other mods or plugins installed including their versions
 * your :doc:`logs`
 
- * please use a service like `GitHubs gist <https://gist.github.com/>`_ or `pastebin <http://pastebin.com/>`_ to provide
+ * please use a service like `GitHubs gist <https://gist.github.com/>`_ or `pastebin <https://pastebin.com/>`_ to provide
```

Comment:
The SpongeForge issue template discourage the use of pastebin/hastebin. See: https://github.com/SpongePowered/SpongeForge/commit/488af7ca45fa054eea7ba066ed12d401a2ab5bca 
This should probably be updated to reflect those changes.